examples/waspleau/.wasproot

```diff
@@ -0,0 +1 @@
+File marking the root of Wasp project.
```

Comment:
We could also maybe add something like "Please never touch/delete this manually", or sth like that.

Reply:
Yeah I can create an issue for this update if you want!